### PR TITLE
Migrate repository to match new Starter Kit structure

### LINKED ISSUE #480: Update to new XMC starter structure
Once the [PR](https://github.com/sitecorelabs/xmcloud-foundation-head-dev/pull/170) on the Foundation head repo for the new structure is merged. This repo needs to be updated to match the new structure.

### PR DESCRIPTION
**Pull request by robearlam**
This PR will update the structure of the repo to match the new default structure defined in the XMC starter kits. It will also change the repo to match FED-First development,  again matching the XMC Starter Kit funcitonality.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] I have read the Contributing guide.
- [x] My code/comments/docs fully adhere to the Code of Conduct.
- [x] My change is a code change.
- [ ] My change is a documentation change and there are NO other updates required.

Closes #480

### CODE REVIEW

**Review by sc-ivanlieckens — CHANGES_REQUESTED** [review]

File: docker/build/cm/templates/import-templates.ps1 [hunk 1]
```diff
@@ -1,33 +0,0 @@
-param(
```
[the remark this answers is not in the document]
Comment (robearlam) [reply docker/build/cm/templates/import-templates.ps1:1]: We dont use this in our scripts

File: docker/build/rendering/Development.ps1 [hunk 2]
```diff
@@ -1,158 +0,0 @@
-# Homemade dotnet "watch" script, watches the deploy folder for changes to assembly files.
```
[the remark this answers is not in the document]
Comment (robearlam) [reply docker/build/rendering/Development.ps1:1]: Yeah, I migrated it to match the `dotnet watch` approach you create for the ASP.NET Core StarterKit

File: README.md [hunk 3]
```diff
@@ -116,64 +40,71 @@ You can follow these steps to create a new XM Cloud Project & Environment, then
...
-
-## 💻+🌏 Running in Edge Development Mode without Docker
-
-### Initialize your repository for Edge Development Mode without Docker
-First initialize your repo using the `.init/ps1` script, you will need to pass in the `-Edge_Token` & `-EdgeNoDocker` parameters to ensure that the repository is correct initialized.
-
-```ps1
-.\init.ps1 -InitEnv -Edge_Token "<<YOUR_EDGE_TOKEN>>" -Edge_NoDocker
+## Running the MVP Site
+To run the MVP site you will need to populate the `./headapps/MvpSite/Mvp.Project.MvpSite.Rendering/appsettings.Development.json`. You will need to provide the Sitecore instance, and Okta sections, it's completed it look something like:
```
[the remark this answers is not in the document]
Comment (IvanLieckens) [reply README.md:47]: Srry this thread was an accidental double post due to the slowness of the review system. The changeset is too large for github to handle 😆

[the remark this answers is not in the document]
Comment (robearlam) [reply README.md:47]: Yeah I hear you, but .NET doesn't work with `.env` files OOTB, so we'd need to write this functionality into the configuration parsing on app start?

[the remark this answers is not in the document]
Comment (robearlam) [reply README.md]: Removed appsettings.Development.json and excluded it in the .gitignore. Updated the readme to have the developert create and populate that file.

**Review by sc-ivanlieckens — APPROVED** [review]

**Pull request merged**